corehq/apps/case_search/templates/case_search/case_search.html

```diff
@@ -0,0 +1,186 @@
+{% extends 'style/bootstrap3/base_section.html' %}
+{% load hq_shared_tags %}
+{% load compress %}
+{% load i18n %}
+{% block stylesheets %}
+    <link rel="stylesheet" href="{% new_static 'codemirror/lib/codemirror.css' %}"/>
+    <link rel="stylesheet" href="{% static 'codemirror/addon/fold/foldgutter.css' %}"/>
+{% endblock stylesheets %}
+{% compress js %}
+{% block js %}{{ block.super }}
+    <script src="{% new_static 'codemirror/lib/codemirror.js' %}"></script>
+    <script src="{% new_static 'codemirror/mode/javascript/javascript.js' %}"></script>
+    <script src="{% new_static 'codemirror/addon/fold/foldcode.js' %}"></script>
+    <script src="{% new_static 'codemirror/addon/fold/foldgutter.js' %}"></script>
+    <script src="{% new_static 'codemirror/addon/fold/brace-fold.js' %}"></script>
+{% endblock js %}
+{% endcompress %}
+{% block js-inline %}
+    <script>
+     var CaseSearchModel = function(){
```

Comment:
@dimagi/js-team is something of this size recommended to be moved to an external JS file?